hackathon_site/dashboard/frontend/src/components/dashboard/TeamCardAdmin/TeamCardAdmin.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import styles from "./TeamCardAdmin.module.scss";
+import Container from "@material-ui/core/Container";
+import Typography from "@material-ui/core/Typography";
+import Card from "@material-ui/core/Card";
+
+import TitledPaper from "components/general/TitledPaper/TitledPaper";
+
+interface CardProps {
+    team_name?: string;
```

Comment:
Could you change this prop name to `teamCode` only bc in the backend that's what we call it and better to be consistent.

Also, is there a reason why these props are optional? In the database a team will always have a team code and members can take in an empty array so I think you can make these required props.

Reply:
I changed the propname to teamCode and I made the teamCode and members props non-optional